website/static/js/issue.js

```diff
@@ -77,6 +77,8 @@ $(function () {
         var old_message = $('#text_div_'+comment_id).text();
         $('#text_comment_'+comment_id).val(old_message);
         $('#text_div_'+comment_id).text('');
+        $(this).hide();
+        $('#del_comment_'+comment_id).hide();
```

Comment:
This is the wrong way!
Remove the id from delete button.
Replace this `$('#del_comment_'+comment_id).hide();` with `$(this).closest('.del_comment').hide();`